scripts/usb_reset.py

```diff
@@ -14,46 +14,273 @@
 # along with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 #
-# ./usb_reset.py 1-2
-# reset USB device 1-2
+# attach
+# ./usb_reset.py attach device -d dom-id -p pid [-r]
+# ./usb_reset.py attach 2-2 -d 12 -p 4130
+# ./usb_reset.py attach 2-2 -d 12 -p 4130 -r
+# 1. reset device
+# if without -r, do step 2~4
```

Comment:
you may need one comment line to explain the [-r] usage?

Reply:
it's explained at line 22 and 69